### PR TITLE
Config edit command

### PR DESCRIPTION
**Pull request by omero**

### CODE REVIEW

File: src/Command/ConfigEditCommand.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Drupal\Component\Serialization\Yaml;
+use Symfony\Component\Process\Process;
+use Symfony\Component\Yaml\Parser;
+use Symfony\Component\Filesystem\Exception\IOExceptionInterface;
+use Symfony\Component\Process\ProcessBuilder;
```
Comment (dmouse) [inline src/Command/ConfigEditCommand.php]: Foul add this dependence composer.json

File: src/Command/ConfigEditCommand.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+class ConfigEditCommand extends ContainerAwareCommand
+{
+
+    protected function configure()
```
Comment (dmouse) [inline src/Command/ConfigEditCommand.php]: Foul add docblock

File: src/Command/ConfigEditCommand.php [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    {
+        $this
+          ->setName('config:edit')
+          ->setDescription($this->trans('commands.config.edit.description'))
+          ->addArgument('config-name', InputArgument::REQUIRED,
+            $this->trans('commands.config.edit.arguments.config-name'));
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
```
Comment (dmouse) [inline src/Command/ConfigEditCommand.php]: Foul add docblock

File: src/Command/ConfigEditCommand.php [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+            $config->save();
+        }
+
+        echo $process->getOutput();
+    }
+
+    /**
+     * @param $config_name    String
+     */
+    protected function getYamlConfig($config_name)
```
Comment (dmouse) [inline src/Command/ConfigEditCommand.php]: Foul add return on the docblock

**Comment by jmolivas** [comment]
Thanks for the PR @omero

**Pull request merged**